src/App.js

```diff
@@ -35,11 +61,14 @@ class App extends React.Component {
           <ScrollToTop>
             <div className="flyout">
               <Navbar darkMode={this.state.darkMode} />
-              <main>
-                <Routes globalProps={{ ...this.state }} />
+              <main className={this.state.darkMode ? "darkMode" : undefined}>
+                <Routes
+                  mode={this.state.darkMode}
```

Comment:
The mode is no longer required to be passed on because everything happens within the App.js and Footer.